### PR TITLE
resolve peer review issue regarding `create_sampling_hist` and `plot_sampling_hist` functions

### PR DESCRIPTION
**Pull request by YueJiangMDSV**
Hi Team,

This pr should resolve peer review issue regarding `create_sampling_hist` and `plot_sampling_hist` functions  #82 

I updated the NAMESPACE file. deleted unused argument, checked input dataframe columns, and changed the input name in consistent with other functions. 

Thanks!

Alex

### CODE REVIEW

File: tests/testthat/test-create_sampling_his.R [hunk 1]
```diff
@@ -5,28 +5,49 @@ n_s_test1 <- c('data', 10)
...
+})
 
 test_that('should throw error', {
-  expect_error(create_sampling_hist(samples_test, Variable, n_s, 3), 'Samples should be a data frame or a tibble')
+  expect_error(create_sampling_hist(test_df1, Variable, n_s), "The input samples dataframe should have contain 'replicate', 'size', and 'rep_size' columns")
 })
 
 test_that('should throw error', {
-  expect_error(create_sampling_hist(samples, data, n_s, 3), "Variable must be a column in 'samples' df")
+  expect_error(create_sampling_hist(test_df2, Variable, n_s), "The input samples dataframe should have contain 'replicate', 'size', and 'rep_size' columns")
```
Comment (hwilliams10) [inline tests/testthat/test-create_sampling_his.R]:
```suggestion
  expect_error(create_sampling_hist(test_df2, Variable, n_s), "The input samples dataframe should contain 'replicate', 'size', and 'rep_size' columns")
```

File: tests/testthat/test-create_sampling_his.R [hunk 2]
```diff
@@ -5,28 +5,49 @@ n_s_test1 <- c('data', 10)
...
+  expect_error(create_sampling_hist(samples_test, Variable, n_s), 'Samples should be a data frame or a tibble')
+})
+
+test_that('should throw error', {
+  expect_error(create_sampling_hist(samples, data, n_s), "Variable must be a column in 'samples' df")
+})
 
 test_that('should throw error', {
-  expect_error(create_sampling_hist(samples_test, Variable, n_s, 3), 'Samples should be a data frame or a tibble')
+  expect_error(create_sampling_hist(test_df1, Variable, n_s), "The input samples dataframe should have contain 'replicate', 'size', and 'rep_size' columns")
```
Comment (hwilliams10) [inline tests/testthat/test-create_sampling_his.R]:
```suggestion
  expect_error(create_sampling_hist(test_df1, Variable, n_s), "The input samples dataframe should contain 'replicate', 'size', and 'rep_size' columns")
```

File: tests/testthat/test-create_sampling_his.R [hunk 3]
```diff
@@ -5,28 +5,49 @@ n_s_test1 <- c('data', 10)
...
 })
 
 test_that('should throw error', {
-  expect_error(create_sampling_hist(samples, data, n_s, 3), "Variable must be a column in 'samples' df")
+  expect_error(create_sampling_hist(test_df2, Variable, n_s), "The input samples dataframe should have contain 'replicate', 'size', and 'rep_size' columns")
 })
 
 test_that('should throw error', {
-  expect_error(create_sampling_hist(samples, Variable, n_s, rep_test), "Number of replications should be a numerical value, a vector with length 1")
+  expect_error(create_sampling_hist(test_df3, Variable, n_s), "The input samples dataframe should have contain 'replicate', 'size', and 'rep_size' columns")
```
Comment (hwilliams10) [inline tests/testthat/test-create_sampling_his.R]:
```suggestion
  expect_error(create_sampling_hist(test_df3, Variable, n_s), "The input samples dataframe should contain 'replicate', 'size', and 'rep_size' columns")
```

File: tests/testthat/test-create_sampling_his.R [hunk 4]
```diff
@@ -5,28 +5,49 @@ n_s_test1 <- c('data', 10)
...
 })
 
 test_that('should throw error', {
-  expect_error(create_sampling_hist(samples, Variable, n_s_test1, 100), "Samples' sizes should be a list or a vector with only numeric values")
+  expect_error(create_sampling_hist(samples, Variable, n_s_test1), "Samples' sizes should be a list or a vector with only numeric values")
```
Comment (hwilliams10) [inline tests/testthat/test-create_sampling_his.R:46]:
```suggestion
  expect_error(create_sampling_hist(samples, Variable, n_s_test1), "Samples' sizes should be a list or a vector with only numeric values")
```

**Review by hwilliams10 — COMMENTED** [review]
Just a couple minor grammatical suggestions. Please let me know if you agree.

File: tests/testthat/test-create_sampling_his.R (hunk 2 above)
Comment (YueJiangMDSV) [reply tests/testthat/test-create_sampling_his.R]: Thanks for pointing out!

File: tests/testthat/test-create_sampling_his.R (hunk 4 above)
Comment (YueJiangMDSV) [reply tests/testthat/test-create_sampling_his.R:46]: Thanks for pointing out!

**Review by tguo9 — APPROVED** [review]
Looks great! Nice work @YueJiangMDSV! Thanks @hwilliams10's suggestions!

**Pull request merged**